server/internal/app/auth_client.go

```diff
@@ -53,6 +54,14 @@ func authMiddleware(cfg *ServerConfig) echo.MiddlewareFunc {
 	}
 }
 
+func jwtEchoMiddleware(cfg *ServerConfig) echo.MiddlewareFunc {
+	mw, err := appx.AuthMiddleware(cfg.Config.JWTProviders(), adapter.ContextAuthInfo, true)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
```suggestion
	mw := lo.Must(appx.AuthMiddleware(cfg.Config.JWTProviders(), adapter.ContextAuthInfo, true))
```

I used `lo.Must` to summarize it in one line, was it hard to read so that you created a new function?

Reply:
I thought that it looks cleaner this way.
Anyway I will use `lo.Must` to summarize it as you mentioned above. 
Thank you!